ot3_state_manager/generate_state_manager_headers.py

```diff
@@ -0,0 +1,62 @@
+"""Script to generate hpp file with OT3StateManager message enums."""
+
+import argparse
+import sys
+from enum import Enum
+from typing import Type
+
+from header_generation_utils.utils import generate_file_comment, write_enum_cpp
+
+from ot3_state_manager.messages import MessageID
+from ot3_state_manager.util import Direction, MoveMessageHardware, SyncPinState
+
+
+def get_message_id_enum() -> Type[Enum]:
```

Comment:
why do we need to do this? isn't `MessageId` already an enum?

Reply:
See comments added in following commit
21aaf6cbb6418fb6650ce0ab1fd8fbd38e470935